src/tools/rbd_mirror/ImageReplayer.h

```diff
@@ -142,7 +142,7 @@ class ImageReplayer {
   bool on_start_interrupted();
```

Comment:
Let's also change `JOURNAL_REPLAY_SHUT_DOWN` to `REPLAY_SHUT_DOWN` in the ASCII art a few lines.

Reply:
Done.